AdyenCard/Components/Card/CardScannerController.swift

```diff
@@ -20,13 +20,6 @@ internal protocol CardScannerProviding {
 }
 
 internal protocol CardScannerControlling: CardScannerAvailability {
-
-    init(
```

Comment:
@nauaros 
Having this defined on the interface level typically creates unnecessary load when it comes down to juggling around with various entities conforming to a specific protocol. It also exposes some implementation details and dependencies on the interface level.